.github/ISSUE_TEMPLATE/maintenance.yml

```
@@ -0,0 +1,63 @@
+name: Maintenance
+description: Suggest an activity to help address tech debt, governance, and anything internal
+title: "Maintenance: TITLE"
+labels: ["internal", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to help us improve operational excellence.
```

Same. What about:
(nit) I think "operational excellence" is a jargon mostly used by AWS SAs. It might not be very clear to readers. What if we make it more generic?

```suggestion
        Thank you for taking the time to help us improve this project.
```